### DISCUSSION TITLE
Difference between Asteroid one and framework

### DISCUSSION THREAD

**Discussion by N6REJ** [templaza/astroid-framework#489] [Q&A] [answered]
You show 3 different downloads but don't explain what their difference are?  What are they?

**Comment by paurray** [comment]
Work in progress but maybe this helps: https://github.com/templaza/astroid-framework/discussions/487

**Comment by Chacapamac** [comment, marked as answer]
@N6REJ 
The **Framework** is the **Astroid motor**. **Astroid One** & **Zero** are the **frontend templates** _(skins)_ that you can attach to that motor.
